app/assets/javascripts/shared/intl_tel_input_validation.js

```diff
@@ -0,0 +1,35 @@
+// validate the telephone number on the blur event
+
+var TelInputValidation = function(input_id, error_msg_id, error_messages) {
```

Comment:
Javascript tarafında StandardJS standartlarını kullanıyoruz (Javascript'in Rubocop'u), örneğin bu standartta satır sonu noktalı virgül yok.  Nasıl kullanılacağını ayrıca anlatırım.  Offense'leri düzeltiyorum.

Reply:
Komit yaptım, bu haliyle elle test edersen iyi olur.

Reply:
@ecmelkytz elle testin tamamlandığında sıkıntı yoksa haber verirsin.

Reply:
Test ettim bir sıkıntı yok.